sbt

```diff
@@ -18,15 +19,27 @@ do
       shift
     fi
     if [ "$arg" == "--team-city" ]; then
-      f=$(grep '^teamcity.configuration.properties.file' "$TEAMCITY_BUILD_PROPERTIES_FILE" | cut -d'=' -f 2)
-      TC_PARAMS="-Dteamcity.configuration.properties.file=$f"
-      echo "Adding team city property: $TC_PARAMS"
+      echo "Ignoring '--team-city' in favour of checking for TEAMCITY_BUILD_PROPERTIES_FILE env var; please remove this switch"
```

Comment:
Given we control both ends of this, do we need this deprecation step? Why not just drop it?

Reply:
Just being careful/considerate, really.  We need builds to continue until this is merged.  Then I'd remove the switch from TC, then remove from here.